lib/generators/active_admin/resource/templates/admin.rb

```diff
@@ -1,5 +1,18 @@
 ActiveAdmin.register <%= class_name %> do
 
+# Available Filters
+#<% class_name.constantize.new.attributes.keys.each do |attr| %>
+# filter :<%= attr.gsub(/_id$/, '') %><% end %>
```

Comment:
1. There should be a way without initialize a instance to get the attributes
2. The code style can be nicer:
```ruby
<% class_name.constantize.new.attributes.keys.each do |attr| -%>
# filter :<%= attr.gsub(/_id$/, '') %>
<% end -%>
```